modules/cloud-init/variables.tf

```diff
@@ -0,0 +1,74 @@
+variable "airgap_package_url" {
+  default     = ""
+  description = "The URL of an airgap package which contains a TFE release."
+  type        = string
+}
+
+variable "airgap_installer_url" {
+  default     = "https://install.terraform.io/installer/replicated-v5.tar.gz"
+  description = "The URL of an airgap package which contains the cluster installer."
+  type        = string
+}
+
+variable "application_config" {
+  description = "The application configuration."
+  type        = map(map(string))
+}
+
+variable "custom_ca_cert_url" {
+  default     = ""
+  description = "The URL of a certificate authority bundle which contains custom certificates to be trusted by the application."
+  type        = string
+}
+
+variable "distribution" {
+  default     = "ubuntu"
+  description = "The type of Linux distribution which will be running on the machines."
+  type        = string
+}
+
+variable "license_file" {
+  description = "The pathname of a Replicated license file for the application."
+  type        = string
+}
+
+variable "proxy_address" {
+  description = "The IP address of the Kubernetes API."
```

Comment:
This feels ambiguous between the variable name and the description, can we add more detail on what we mean here?

ie, is this the externally routable IP of the Kubernetes API behind a load balancer? or is this any one of the primary addresses? or is this the internal pod ip address of the kubelet? 

We could either add more description or change the variable name to be more descriptive

Reply:
Fixed by https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/58/commits/58931f1d77fe8a0345d4333c70b4c2703c9ce5a9 and https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/58/commits/53f3de877d72adb0214f006d84f2edea8944be8b.